lib/ansible/module_utils/facts/hardware/linux.py

```diff
@@ -485,6 +488,8 @@ def get_mount_facts(self):
         pool = ThreadPool(processes=min(len(mtab_entries), cpu_count()))
         maxtime = globals().get('GATHER_TIMEOUT') or timeout.DEFAULT_GATHER_TIMEOUT
         for fields in mtab_entries:
+            # Transform octal escape sequences
+            fields = [self.MTAB_FIELD_OCTAL_ESCAPE_RE.sub(lambda x: chr(int(x.group()[1:], 8)), field) for field in fields]
```

Comment:
Can this be rewritten using a function rather than a lambda for easier maintenance going forward? This is a pretty dense line.

Reply:
It could. It just didn't seem necessary with it only being used on one place.

Reply:
From our triage meeting, we all found we were confused by this line.  Making it more verbose will likely make it easier to understand.

Reply:
Done.